FrontEnd/src/components/PodPage.js

```diff
@@ -1,13 +1,14 @@
-import React, {Component} from 'react'
+import React, { Component } from 'react';
 
 import PodRules from './PodRules';
 import PodDataset from './PodDataset.js';
 
 import * as StudyService from "../studyService"
 
 import Sepes from '../sepes.js';
+import Nsgswitch from './nsgSwitch';
 const sepes = new Sepes();
-
+var self = this;
```

Comment:
This one is dangerous hope that self is not used. Do't even know what self referees to but nothing good.
The "let self = this" should only be used methods of classes. This line must be removed.

Reply:
It was a leftover of an attempted bug fix where a function/class did not get sent in the correct context to function correctly.
It was unused in the code